sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/plans/logical/statsEstimation/EstimationUtils.scala

```diff
@@ -71,6 +71,15 @@ object EstimationUtils {
     AttributeMap(output.flatMap(a => inputMap.get(a).map(a -> _)))
   }
 
+  /** Match alias with its child's column stat */
+  def getAliasStats(expressions: Seq[Expression],
```

Comment:
nit: indent (also the following line, please see the other methods)

Reply:
Done